sr_description/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!--
  Software License Agreement (BSD License)
- Copyright © 2022 belongs to Shadow Robot Company Ltd.
+ Copyright © 2022, 2024 belongs to Shadow Robot Company Ltd.
```

Comment:
do we need this change?

Reply:
I've given the CI a stern talking to, but apparently yes